src/foam/u2/tag/Select.js

```diff
@@ -20,23 +20,26 @@ foam.CLASS({
   name: 'Select',
   extends: 'foam.u2.View',
 
+  implements: [
+    'foam.u2.ChoiceViewTrait'
```

Comment:
As mentioned for the Input tag changes, should we really be adding this functionality to the Select tag?

Things under foam.u2.tag, are supposed to just represent raw tags, and I feel like more advanced stuff such as making a choice view that accepts choices or choiceDao should be part of a higher level component.